std/experimental/checkedint.d

```diff
@@ -980,6 +985,17 @@ if (isIntegral!T || is(T == Checked!(U, H), U, H))
         x += 1;
         assert(MyHook.thereWereErrors);
     }
+
+    @system unittest
+    {
+        auto x1 = Checked!int(10);
+        auto x2 = Checked!int(10);
+        x1 += x2;
+        assert(x1.get == 20);
+        auto x3 = Checked!(Checked!int)(10);
+        x1 += x3;
+        assert(x1.get == 30);
+    }
```

Comment:
Sadly this needs to be outside of the template as otherwise it would be instantiated for every template.
That's the reason for `static if (is(T == int) && is(Hook == void)) unittest` btw. 
Also we typically add a link to the respective bugzilla issue like:

```
// https://issues.dlang.org/show_bug.cgi?id=18142
unittest
...
```